platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/command/ProfileInstallCommand.java

```diff
@@ -0,0 +1,195 @@
+    @Argument(index = 0, name = "profileName", description = "Name of install profile to use", required = true, multiValued = false)
+    String profileName;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ProfileInstallCommand.class);
+
+    @Override
+    protected void doExecute(ApplicationService applicationService,
+                             FeaturesService featuresService, BundleService bundleService) throws Exception {
+
+        if (profileName.startsWith(".") || profileName.startsWith("/")) {
+            throw new IllegalArgumentException("Profile Name must not start with '.' or '/'");
+        }
+
+        Optional<Feature> optionalProfile = getProfile(applicationService, profileName);
+        Optional<Map<String, List<String>>> optionalExtraProfiles = getProfile(profileName);
+
+        if (optionalProfile.isPresent()) {
+            List<String> profileApps = optionalProfile.get().getDependencies().stream()
+                    .map(Dependency::getName)
+                    .collect(Collectors.toList());
+            startApps(applicationService, profileApps);
+            uninstallInstallerModule(featuresService);
+            printSuccess("Installation Complete");
+        } else {
+            if (optionalExtraProfiles.isPresent()) {
+                Map<String, List<String>> profile = optionalExtraProfiles.get();
+                startApps(applicationService, profile.get(ADVANCED_PROFILE_START_APPS));
+                installFeatures(featuresService, profile.get(ADVANCED_PROFILE_INSTALL_FEATURES));
+                uninstallFeatures(featuresService, profile.get(ADVANCED_PROFILE_UNINSTALL_FEATURES));
+                /* The stop-bundles operation is a workaround currently in place for dealing with
+                    the inter-dependencies of some features, this will likely be removed in the future. */
+                stopBundles(bundleService, profile.get(ADVANCED_PROFILE_STOP_BUNDLES));
+                uninstallInstallerModule(featuresService);
+                printSuccess("Installation Complete");
+            } else {
+                printError(String.format("Profile: %s not found", profileName));
+            }
+        }
+    }
+
+    private void startApps(ApplicationService applicationService, List<String> startupApps)
+            throws ApplicationServiceException {
+        if (startupApps != null) {
+            printSectionHeading("Starting Applications");
+            startupApps.stream().distinct().forEach(application -> {
+                printItemStatusPending("Starting: ", application);
+                executeAsSystem(() -> {
+                    try {
+                        applicationService.startApplication(application);
+                    } catch (ApplicationServiceException e) {
+                        printItemStatusFailure("Start Failed: ", application);
+                        throw new ApplicationServiceException(e);
```

Comment:
Why do we throw a new instance of ApplicationServiceException with the original one as a cause and not just rethrow the original one?